src/typegenFormatPrettier.ts

```diff
@@ -1,10 +1,13 @@
 import path from "path";
+import type * as Prettier from "prettier";
```

Comment:
This syntax required an update of prettier.

Upgrading prettier lead to some new formatting seen throughout this PR. Luckily not much.